docs/configurations.md

```diff
@@ -234,16 +288,28 @@ scalar.db.username=<USERNAME>
 scalar.db.password=<PASSWORD>
 ```
 
-### Example 2
+### Example - App, ScalarDB Server, and database
```

Comment:
Is there a different example that we can give here for this doc on the `master`, 3.10`, and `3.9` branches since ScalarDB Server is deprecated as of 3.9?

Reply:
I think we can use ScalarDB Cluster in this example, but it's for the enterprise edition. So if we don't want to use it here, we might be able to remove this whole example section.

Reply:
That's true... I've removed this part to avoid confusion since this doc applies to Community edition users as well.